src/providers/dataProvider/index.ts

```diff
@@ -100,11 +101,21 @@ export const getDataProvider = async (): Promise<DataProvider<string>> => {
         )
         return record
       },
-      afterCreate: async (record: CreateResult<Project>) => {
+      afterCreate: async (
+        record: CreateResult<Project>,
+        dataProvider: DataProvider
+      ) => {
+        const { data } = record
+        const { id } = data
         await audit(
           AuditType.CREATE_PROJECT,
           `Project created (${String(record.data.id)})`
         )
+        await dataProvider.update<Project>('projects', {
+          id,
+          previousData: data,
+          data: { created_at: DateTime.now().toFormat('yyyy-MM-dd') }
```

Comment:
@TahaKhanAbdalli - could you put the data creation into a reusable function?  Then, if we decide we want to introduce time too, we only have to do it in one location.